src/utils/localStorage.js

```diff
@@ -0,0 +1,25 @@
+const keyAccountConfirmed = (accountId, networkId) => `wallet.account:${accountId}:${networkId}:confirmed`
+
+export const setAccountConfirmed = (accountId, networkId, confirmed) => {
+    try {
+        localStorage.setItem(keyAccountConfirmed(accountId, networkId), confirmed)
+    } catch (err) {
```

Comment:
I think we shouldn't ignore errors here, makes it hard to notice if things went wrong. If we don't have local storage working – we won't be able to store keys for user anyway.

Reply:
Done. I noticed that if localStorage is disabled the Wallet is not loading at all (only console error is shown), maybe we should handle it by adding a page with info for user?

Reply:
@marcinbodnar Good point! I'd expect the % of users with localStorage disabled to be low, so perhaps using a modal to display "You must enable Cookies in order to use this application" is enough

Reply:
@kcole16 we can, but the thing is that local storage availability should be checked as first thing, in this file (if we don't want to complicate stuff too much):
https://github.com/nearprotocol/near-wallet/blob/master/src/index.js
So there will be not much to show undernith modal element.

Please add issue for this, I will think about it.

Reply:
@kcole16 @marcinbodnar I think probably makes sense to have static html landing for cases when either some basic tech we need (like localStorage) is disabled or JS altogether is disabled. This also can provide useful content for web crawlers, etc

Reply:
Sounds good to me. Maybe also some basic info/links on how to change it.